kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -235,8 +235,14 @@ protected void configureMarshaller(Set<Class<?>> classes, final ClassLoader clas
         if (classes == null) {
             classes = new HashSet<Class<?>>();
         }
+
         // add byte array handling support to allow byte[] to be send as payload
         classes.add(JaxbByteArray.class);
+        classes.add(java.time.LocalDateTime.class);
```

Comment:
Is this safe with regards to the if clause below `if (!formatDate) {` ? I would suspect that date types would have same handling in this respect (just guessing though).